control-plane/agents/src/bin/core/volume/specs.rs

```diff
@@ -728,11 +731,14 @@ impl ResourceSpecsLocked {
         } else {
             // on a different node, so connect via an nvmf target
             let mut replica = self.replica(&replica_state.uuid).await?;
-            match replica.share(registry, &replica_state.into()).await {
-                Ok(uri) => Ok(uri.into()),
+            let host_nqn = HostNqn::new_ebs_name(nexus_node.as_str());
```

Comment:
Question: Is it suppposed to be the replica node rather than nexus node? From your example, `super-sayan` is where the replica is right?

Reply:
For multiple replicas on node `super-sayan-1`, `super-sayan-2`, `super-sayan-3`, and nexus on `super-sayan-1`, the replica uri's would be like, 

`nvmf://10.1.0.9:8420/nqn.2019-05.io.openebs:rep-2-uuid?uuid=rep-2-uuid&hostnqn=nqn.2019-05.io.openebs:node-name:super-sayan-2`, 

`nvmf://10.1.0.10:8420/nqn.2019-05.io.openebs:rep-3-uuid?uuid=rep-3-uuid&hostnqn=nqn.2019-05.io.openebs:node-name:super-sayan-3` 

respectively right?  Is my understanding correct?

Reply:
No, the allowed-host is where the nexus lives. Because it's the nexus that connects to the replica.
So would be:

`nvmf://10.1.0.9:8420/nqn.2019-05.io.openebs:rep-2-uuid?uuid=rep-2-uuid&hostnqn=nqn.2019-05.io.openebs:node-name:super-sayan-1`

`nvmf://10.1.0.10:8420/nqn.2019-05.io.openebs:rep-3-uuid?uuid=rep-3-uuid&hostnqn=nqn.2019-05.io.openebs:node-name:super-sayan-1`

Reply:
Okay. Understood.